about/blog/body.html.erb

```diff
@@ -8,19 +8,47 @@
     <% topic_list(opts: { category: 'blog', no_definitions: true, per_page: 10, page: params[:page] }).each.with_index do |topic, index| %>
       <%= render partial: "topic_list_item", locals: {
         topic: topic,
-        topic_item_classes: "tag-getting-started #{index == 0 ? "post-card-large" : ""}",
+        item_classes: "tag-getting-started #{index == 0 ? "post-card-large" : ""}",
         topic_excerpt_length: index == 0 ? 30 : 20,
+        topic_url: "/blog/#{topic.id}",
         user_profile_opts: {
           include_avatar: true,
           avatar_size: 60
         }
       } %>
     <% end %>
   </div>
+</div>
 
-  <div class="controls right">
-    <a class="btn page-prev">Previous<%= SvgSprite.raw_svg('angle-up') %></a>
-    <a class="btn page-next">Next<%= SvgSprite.raw_svg('angle-down') %></a>
-  </div>
+<div class="subscribe">
+  <% if current_user %>
+    <div class="subscribe-user-button">
+      <img width="35" height="35" class="subscribe-avatar" src="<%= current_user.small_avatar_url %>">
+      <div class="subscribe-label">Subscribe</div>
+    </div>
+  <% else %>
+    <div class="subscribe-guest-button"></div>
+  <% end %>
+</div>
 
-</div>
\ No newline at end of file
+<% if current_user %>
+  <div class="subscribe-modal">
+    <div class="subscribe-modal-content">
+      <div class="modal-header">
+        <span class="subscribe-modal-close">&times;</span>
+        <h3>Subscribe!</h3>
+      </div>
+      <div class="modal-content">
+        <div class="subscribe-checkbox">
+          <span>Subscribe to blog</span>
+          <input type="checkbox" class="checkbox" checked=<%= current_user.present? %>/>
+        </div>
+      </div>
+      <div class="modal-footer">
+        <div class="subscribe-buttons">
+          <button type="button" class="btn btn-secondary">Submit</button>
+        </div>
```

Comment:
@MeghnaAJ Let's also add a ``Cancel`` link here:

```
<a class="cancel-link">Cancel</a>
```

Reply:
I have made this change and will send a separate PR for "subscribe" HTML once this PR is merged.